### ISSUE TITLE
Don't store full filenames in ssr-manifest.json and main.js

### ISSUE THREAD

**Issue by IlyaSemenov** [frandiox/vite-ssr#19]
Currently, `ssr-manifest.json` stores _full local_ filenames:

```
{
  "/Users/semenov/work/xxx/node_modules/@vue/shared/dist/shared.esm-bundler.js": [
    "/assets/vendor.8b10200d.js"
  ],
  "/Users/semenov/work/xxx/node_modules/@vue/reactivity/dist/reactivity.esm-bundler.js": [
    "/assets/vendor.8b10200d.js"
  ],
```

`main.js` also has this:

```
./server/main.js:  (ssrContext.modules || (ssrContext.modules = new Set())).add("/Users/semenov/work/xxx/src/pages/index.vue");
./server/main.js:  (ssrContext.modules || (ssrContext.modules = new Set())).add("/Users/semenov/work/xxx/src/App.vue");
```

* That is not safe from the privacy point of view.
* That most probably means that the package is not redistributable (I am yet to try, but I wouldn't be surprised if it won't work)

**Comment by frandiox** [comment]
@IlyaSemenov `ssr-manifest.json` is created by Vite itself so I think this is something that can be discussed in Vite's repo instead.

To my understanding, the paths are not really used as real "paths", but as "unique identifiers". Therefore, it shouldn't matter which machine runs them. Therefore, when the server renders a page with a dependency on `User/path/xxx.js`, it will map it using the ssr-manifest JSON to `assets/xxx.js`.

**Comment by frandiox** [comment]
There's a PR for this in Vite repo: https://github.com/vitejs/vite/pull/3315

**Comment by frandiox** [comment]
Seems to be fixed in Vite `2.4.0-beta.2` 👍
